ome_zarr/scale.py

```diff
@@ -129,7 +130,7 @@ def __nearest(self, plane: np.ndarray, sizeY: int, sizeX: int) -> np.ndarray:
         """Apply the 2-dimensional transformation."""
         return cv2.resize(
             plane,
-            dsize=(sizeY // self.downscale, sizeX // self.downscale),
+            dsize=(sizeX // self.downscale, sizeY // self.downscale),
```

Comment:
Really? Wow! This implies that we would likely need to re-downscale all non-square images, no?

Reply:
Yes, I was surprised too, and googled this - e.g. https://www.tutorialkart.com/opencv/python/opencv-python-resize-image/
```
dim = (width, height)
resized = cv2.resize(img, dim, interpolation = cv2.INTER_AREA)
```
Also, `omero-cli-zarr` has:
```
        plane = cv2.resize(
                            plane,
                            dsize=(size_x // 2, size_y // 2),
                            interpolation=cv2.INTER_NEAREST,
                        )
```

I guess we just haven't used `ome_zarr scale` very much.